### ISSUE TITLE
NTR: Phosphoserine phosphatase low in fibroblast (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9252]
New term request Phosphoserine phosphatase low in fibroblast (elthat_111107051616): PSPH  UMLS: 1291463

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000080
- label: Reduced phosphoserine phosphatase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:9222972
- def: Acitivity of phosphoserine phosphatase below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9252
- synonyms:
